mix.lock

```diff
@@ -1,32 +0,0 @@
-%{
```

Comment:
Why we delete the `mix.lock` ?

Reply:
Since this is an open-source project, `mix.lock` is a little redundant and causes churn in the version control system. Constraints cannot be set from the `mix.lock` for projects.

Therefore, the `mix.lock` build a false sense of security and should not be committed in open source. You want to rely fully on the `mix.exs` file instead. This will also help the project communicate its full version support.

Reply:
@BenMorganIO this is not the case with Elixir projects. Take a peek at some of the official projects, they all have the lock committed:  https://github.com/elixir-ecto/ecto

Including the lock ensure that anyone who uses master gets the _exact_ versions that were known to build.